apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -4904,6 +4904,7 @@
   "unarchive-room": "Unarchive Room",
   "unarchive-room_description": "Permission to unarchive channels",
   "Unassigned": "Unassigned",
+  "unauthorized": "Not authorized",
```

Comment:
didn't we have translations for this one b4? i kinda remember seeing it lol

Reply:
Yeah, we already had another similar translation string i.e. `error-not-authorized`, however, I decided not to use it because there were a lot of test cases like this which were depending on that exact error message. So it seemed a bit simple to just add a new translation, then go around updating all those tests :sweat_smile: 

https://github.com/RocketChat/Rocket.Chat/blob/feature/omnichannel-priorities/apps/meteor/tests/end-to-end/api/00-miscellaneous.js#L576